Core/HLE/sceUsbCam.cpp

```diff
@@ -50,6 +52,17 @@ static int sceUsbCamStopMic() {
 	return 0;
 }
 
+static int sceUsbCamReadMicBlocking(u32 bufAddr, u32 size) {
+	INFO_LOG(HLE, "UNIMPL sceUsbCamReadMicBlocking: size: %d", size);
+	for (unsigned int i = 0; i < size; i++) {
+		if (Memory::IsValidAddress(bufAddr + i)) {
+			Memory::Write_U8(std::rand() & 0xFF, bufAddr + i);
```

Comment:
Would prefer to have emulation behavior consistent between runs.  Since this is just generating test data, can't we just use a constant value?

-[Unknown]